smart-contract/schema/execute_msg.json

```diff
@@ -118,6 +118,26 @@
       },
       "additionalProperties": false
     },
+    {
+      "type": "object",
```

Comment:
@ranadewall new message type supported, we can probably get omnibus to also use this instead of separate tx's

Reply:
👍